xarray/core/variable.py

```diff
@@ -1091,7 +1092,8 @@ def chunk(self, chunks={}, name=None, lock=False):
 
             data = da.from_array(data, chunks, name=name, lock=lock, **kwargs)
 
-        return type(self)(self.dims, data, self._attrs, self._encoding, fastpath=True)
+        new_encoding = None  # rechunking removes all encoding
+        return type(self)(self.dims, data, self._attrs, new_encoding, fastpath=True)
```

Comment:
a simpler way to achieve the same thing is just to omit the argument:
```suggestion
        return type(self)(self.dims, data, self._attrs, fastpath=True)
```